docker/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM ubuntu:18.04
+MAINTAINER Pramod Kumbhar <pramod.s.kumbhar@gmail.com>
+MAINTAINER Matthias Wolf <matthias.wolf@epfl.ch>
+
+ENV DEBIAN_FRONTEND noninteractive
+
+ARG proxy
+ENV http_proxy=$proxy
+ENV https_proxy=$proxy
+
+# default software required
+RUN apt-get update
+RUN apt-get -y install software-properties-common
+RUN add-apt-repository ppa:ubuntu-toolchain-r/test
+RUN apt-get update
+RUN sed -e 's/#.*//g' ../sysconfig/ubuntu-18.04/packages| \
+        xargs -r apt-get --allow-downgrades \
+                         --allow-remove-essential \
```

Comment:
I am mssing something obvious : could you remind me how sysconfig dir available before cloning repo?

Reply:
Sorry, that's why the build fails. Haven't fixed that just yet.